### PR TITLE
Metrics webapp

### PR DESCRIPTION
**Pull request by moise-mulungu**
On this milestone, I:
- Chose a beauty Makeup API for this App
- Created two main pages for my App
- Installed React-redux with their dependencies
- Created components
- Setup the basic structure for my App
- Added functionalities to my main pages
- Created both unit and integration testings.
- Updated the README file 
- Recorded Loom video.

### CODE REVIEW

File: src/__tests__/makeups.test.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import makeupReducer, { getAllFoundations } from '../redux/makeups/makeups';
+
+it('reducer should return list of makeups', () => {
+  const previousState = [];
+  expect(makeupReducer(previousState, getAllFoundations())).toBeDefined();
```
Comment (uwadonat) [inline src/__tests__/makeups.test.js:5]: - [ ] Let's add more unit tests for several components of the app and some pure functions in Redux code.

**Review by uwadonat — CHANGES_REQUESTED** [review]
Hi **@[moise-mulungu](https://github.com/moise-mulungu),** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost


## Highlights
- Your commit and pull request messages are very descriptive ✔️
- No linter errors ✔️
- Your code is well organised and clean ✔️

## Required Changes ♻️

- [ ] Make sure your presentation shows code that you find interesting or you are proud of making.

- [ ] _Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


@moise-mulungu ✋🏼 
#### Great work on making the changes requested by a previous reviewer 👏 
#### You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
___
### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._
- [ ] Like the last reviewer said your presentation should include some code that you are proud of it. You showed only the website. [Requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=you%20should%20also%20highlight%20some%20interesting%20piece%20of%20code%20or%20something%20you%20built%20that%20you%20are%20very%20proud%20of.)
___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
-[ ] Please fix the broken link in the license. It is important for make professional readme.
image

___

### Cheers and Happy coding! 💻 👏

almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by elmar8287 — APPROVED** [review]
Hi, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**